skyportal/handlers/api/photometry.py

```diff
@@ -1,22 +1,14 @@
-import os
-import io
-import base64
-from pathlib import Path
-import tornado.web
+import arrow
```

Comment:
The non-syntax changes to this file are due to the fact that the parent may now either be a comment or a candidate, so we add checks here